### PR TITLE
Update test.txt

### PR DESCRIPTION
**Pull request by hadredess**
add commit "commit 4
commit 5"

### CODE REVIEW

**Comment by R3-da** [comment]
yo @hadredess can you provide more explanations

**Comment by R3-da** [comment]
can you provide more explanations

**Comment by R3-da** [comment]
@hadredess can you provide more explanations

**Comment by R3-da** [comment]
@hadredess test

**Comment by hadredess** [comment]
@R3-da test 1

File: test.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 commit 1
 commit 2
-commit 3
\ No newline at end of file
+commit 3
```
Comment (hadredess) [inline test.txt:3]: i don't know what has changer here

File: test.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+commit 4
```
Comment (hadredess) [inline test.txt:4]: well add commit 4

File: test.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+commit 5
```
Comment (hadredess) [inline test.txt:5]: i don't know about that

Comment (hadredess) [inline test.txt:5]:
```suggestion
commit 56
```
@hadredess

File: test.txt [hunk 4]
```diff
@@ -1,3 +1,5 @@
 commit 1
 commit 2
-commit 3
```
Comment (hadredess) [inline test.txt:3]: i don't know exactly what changed here

**Review by hadredess — COMMENTED** [review]
test 2

**Pull request merged**